lasagne/layers/merge.py

```diff
@@ -68,22 +70,57 @@ class ElemwiseSumLayer(MergeLayer):
     before a :class:`DenseLayer`, insert separate :class:`DenseLayer` instances
     of the same number of output units and add them up afterwards. (This avoids
     the copy operations in concatenation, but splits up the dot product.)
```

Comment:
No, the `ElemwiseSumLayer` is the only way to avoid a `ConcatLayer` in front of a `DenseLayer`, because the matrix product in the `DenseLayer` can be expressed as a sum of some smaller matrix products.